app/views/admin/index.html.erb

```diff
@@ -10,6 +10,18 @@
     </nav>
 </div>
 
+
+<div class="column">
+  <section class="top_customers">
+    <h3>  Top 5 Customers</h3>
+      <ol>
+        <% @top_5_customers.each do |customer| %>
+          <li><%="#{customer.first_name} #{customer.last_name} has made #{customer.result_count} successful transactions"%></li><br>
```

Comment:
potential refactor using the `customer.full_name` instance method Zanna and I put together

Reply:
Also, what is `customer.result_count`? I don't see that as an instance method anywhere.